docs/application/dotnet/get-started/nui/quickstart.md

```diff
@@ -0,0 +1,291 @@
+                animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
+                animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
+                animation.Looping = true;
+                animation.Play();
+            }
+
+            public void OnKeyEvent(object sender, Window.KeyEventArgs e)
+            {
+                if (e.Key.State == Key.StateType.Down && (e.Key.KeyPressedName == "XF86Back" || e.Key.KeyPressedName == "Escape"))
+                {
+                    Exit();
+                }
+            }
+
+            static void Main(string[] args)
+            {
+                var app = new Program();
+                app.Run(args);
+            }
+        }
+    }
+
+```
+#### To create an application that displays the "Hello World" text label
+
+
+1.  Declare the required system and NUI namespaces:
+
+    ```
+    using System;
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    ```
+
+2.  Scope the application to the `SampleNUIApp` namespace:
+
+    ```
+    namespace SampleNUIApp
+    ```
+
+3.  Derive the application from the [Tizen.NUI.NUIApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIApplication.html) class:
+
+    ```
+    class Program : NUIApplication
+    ```
+
+    The `Tizen.NUI.NUIApplication` class includes constructors that allow you to create applications with various stylesheets and window modes.
+
+4.  To handle behavior when the application is launched, override the `OnCreate()` method of the `Tizen.NUI.NUIApplication` class and call the initialization method:
+
+    ```
+    protected override void OnCreate()
+    {
+        base.OnCreate();
+        Initialize();
+    }
+    ```
+
+    > **Note**  
+    > To invoke the `Created` event of the [Tizen.Applications.CoreApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.CoreApplication.html) class, you must call the `base.OnCreate()` method inside the override.
+
+5.  Within the `Initialize()` method, set the text label properties:
+
+    1.  Create the text label object from the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class:
+
+        ```
+        TextLabel text = new TextLabel("Hello Tizen NUI World");
+
+        ```
+
+    2.  Align the text horizontally to the center of the available area:
+
+        ```
+        text.HorizontalAlignment = HorizontalAlignment.Center;
+        ```
+
+    3.  To illustrate the label width, set the label text color:
+
+        ```
+        text.TextColor = Color.Blue;
+        ```
+
+    4.  Define the text size in points:
+
+        ```
+        text.PointSize = 12.0f;
+        ```
+
+    For more information on the key properties of the `Tizen.NUI.BaseComponents.TextLabel` class, see [TextLabel](textlabel.md).
```

Comment:
Broken link. Please change to ../../guides/nui/textlabel.md

Reply:
Modified.